launch-dev

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash -ex
+
+docker rm -f charge-dev || true
+
+docker run --name charge-dev -d \
+   -p 4567:8881 \
+   -e ENABLE_S3_UPLOADS='false' \
+   charge \
+   ruby charge.rb
```

Comment:
You shouldn't need this, right? Because that's already the entrypoint for this container?

Reply:
No biggie.